### ISSUE TITLE
Entry Block does not show minute markers

### ISSUE THREAD

**Issue by carryforward** [FactomProject/WorkItems#624]
I run:
~/go/src/github.com/FactomProject/factom-cli/trans.sh
then
echo "hello" | factom-cli mkchain -e thisisanexternalid jane

1 minute acceleration is turned on
and I get 
```
DB E1 83 45 13 2E 9F 1B D2 48 B7 F4 1D A6 4B D2 FA D1 47 94 52 AD 50 9F A8 A4 B0 0C A3 71 4F CC 
EC 31 DB 84 C8 C1 45 5F F5 E7 67 A2 3C 9B EF A3 5E 48 1E 83 97 48 7C 85 4F 1C CF 66 BE 73 D7 96 
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
00 00 00 00 
00 00 00 6D 
00 00 00 01 
EC 31 DB 84 C8 C1 45 5F F5 E7 67 A2 3C 9B EF A3 5E 48 1E 83 97 48 7C 85 4F 1C CF 66 BE 73 D7 96
```

There is no minute marker for the first entry in the binary.  I would expect:
```
DB E1 83 45 13 2E 9F 1B D2 48 B7 F4 1D A6 4B D2 FA D1 47 94 52 AD 50 9F A8 A4 B0 0C A3 71 4F CC 
6c 91 7b 42 da 6b b9 89 19 bd fa 09 62 fe d7 b1 08 1c 34 3a 73 74 24 77 23 dd a7 7a da aa b0 9c
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
00 00 00 00 
00 00 00 6D 
00 00 00 02 
EC 31 DB 84 C8 C1 45 5F F5 E7 67 A2 3C 9B EF A3 5E 48 1E 83 97 48 7C 85 4F 1C CF 66 BE 73 D7 96
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 06
```

**Comment by carryforward** [comment]
I changed it to 10 minute blocks and see the same problem.  it does not show any minute markers.

**Comment by carryforward** [comment]
probably need to be solved before #511
